ts/components/FiscalCodeLandscapeOverlay.tsx

```diff
@@ -0,0 +1,94 @@
+/**
+ * A component to show the fiscal code fac-simile in Landscape
+ */
+import { Body, Button, Container, Content, View } from "native-base";
+import * as React from "react";
+import { Platform, StatusBar, StyleSheet } from "react-native";
+import { isIphoneX } from "react-native-iphone-x-helper";
+import { UserProfile } from "../../definitions/backend/UserProfile";
+import IconFont from "../components/ui/IconFont";
+import customVariables from "../theme/variables";
+import FiscalCodeComponent from "./FiscalCodeComponent";
+import AppHeader from "./ui/AppHeader";
+
+type Props = Readonly<{
+  onCancel: () => void;
+  profile: UserProfile;
+  showBackSide?: boolean;
+}>;
+
+const globalHeaderHeight: number =
```

Comment:
you could use _Platform.select_
```suggestion
const globalHeaderHeight: number = Platform.select({
  ios: isIphoneX()
    ? customVariables.appHeaderHeight + 42
    : customVariables.appHeaderHeight + 18,
  android: customVariables.appHeaderHeight
});
```

Reply:
updated within https://github.com/teamdigitale/io-app/pull/1228/commits/6f9d3e6aed837b3bd9a901114de05fbebbfd19eb